config/nodemailer.js

```diff
@@ -0,0 +1,9 @@
+import nodemailer from "nodemailer"
+export const transporter= nodemailer.createTransport({
+    service: 'gmail',
```

Comment:
@singhcoder694 Also, use this configuration.

`host: "smtppro.zoho.in",
  secure: true,
  port: 465,
  auth: {
    user: process.env.EMAIL_SENDER,
    pass: process.env.EMAIL_PASSWORD,
  },`

Reply:
Should I copy and paste it

Reply:
smtppro.zoho.in, I don't have account in this

Reply:
Done @ank1traj